### PR TITLE
test: Ensure working E2E test after cobra-integrations

### PR DESCRIPTION
**Pull request by Haruyzal**
**Summary:**

`Fetcher.go` now does not delete the cache-folder every second time we run `go test`.

Merged the Cobra-integration branches in here.

The E2E-test has been changed so it now uses the Cobra-flags.

The E2E now passes.

Resolves. [#72](https://github.com/NovoNordisk-OpenSource/decentralized-tech-radar/issues/72)

### CODE REVIEW

File: test/cache/ForTesting1.csv
Comment (Slug-Boi) [inline test/cache/ForTesting1.csv:1]: Remove cache via cleanup func and also add this to the gitignore

**Review by Slug-Boi — CHANGES_REQUESTED** [review]
See my comments

**Review by zqueamish — APPROVED** [review]

File: src/cache/template1.csv
[the remark this answers is not in the document]
Comment (Haruyzal) [reply src/cache/template1.csv:1]: I cannot see this file in my directory.

File: test/specfile.txt
[the remark this answers is not in the document]
Comment (Haruyzal) [reply test/specfile.txt:1]: Added.

File: test/cache/ForTesting1.csv
Comment (Haruyzal) [reply test/cache/ForTesting1.csv:1]: Added.

File: src/cache/template1.csv
[the remark this answers is not in the document]
Comment (Haruyzal) [reply src/cache/template1.csv:1]: Think I should have deleted it now.

**Review by Slug-Boi — APPROVED** [review]
Looks good now

**Pull request merged**